plugins/ImageActions/main.lua

```diff
@@ -6,6 +6,9 @@ function initUi()
   app.registerUi({["menu"] = "Rotate selected images 90 degree clockwise", ["callback"] = "cb", ["mode"] = 5})
   app.registerUi({["menu"] = "Rotate selected images 90 degree counterclockwise", ["callback"] = "cb", ["mode"] = 6})
   app.registerUi({["menu"] = "Make white pixels transparent", ["callback"] = "cb", ["mode"] = 7})
+  app.registerUi({["menu"] = "Trim selected images", ["callback"] = "cb", ["mode"] = 8})
+  app.registerUi({["menu"] = "Split images vertically by largest white block", ["callback"] = "cb", ["mode"] = 9})
+  app.registerUi({["menu"] = "Split images horizontally by largest white block", ["callback"] = "cb", ["mode"] = 10})
 end
 
 function cb(mode)
```

Comment:
The image splitting is a bit different, so we could use a different callback function. For the other modes we could used named constants instead of numbers.